app/helpers/welcome_helper.rb

```diff
@@ -1,2 +1,23 @@
 module WelcomeHelper
+  NUMBER_OF_SUPPORTED_SERVICES = 2
+
+  def current_user
+    Physiqual::User.find_by_user_id(session[:physiqual_user_id])
+  end
+
+  def logged_in?
+    session[:physiqual_user_id] && current_user
+  end
+
+  def number_of_tokens
+    current_user.physiqual_tokens.size
+  end
+
+  def fitbit_token?
+    current_user.fitbit_tokens.size > 0
```

Comment:
Ja in principe was mijn idee dat je deze functie alleen aan mag roepen als je ingelogd bent. Ik zet er een comment bij dat dat de preconditie is.